css/details.css

```diff
@@ -133,3 +133,16 @@ body {
   gap: 8px;
   flex-wrap: wrap;
 }
+.display-flex{
```

Comment:
Don't name your classes according to the css rules they are applying (`display-flex`).
Name them according to rule they have in the page (`container`, `list-container`, etc.)